azure-pipelines.yml

```diff
@@ -45,3 +49,63 @@ steps:
       kubernetesServiceEndpoint: hellman-mlops-deploy
       command: apply
       arguments: -f grafana/configmaps
+
+  ### Deploy grafana
+  - task: PowerShell@2
+    inputs:
+      targetType: 'inline'
+      script: |
+        kubectl --namespace $(NAMESPACE) create secret generic grafana-password --from-literal=admin-user=admin --from-literal=admin-password=$(ADMIN_PASSWORD) --dry-run=client -o yaml | Out-File k8s/secret.yaml
+
+  - task: Kubernetes@1
+    displayName: kubectl apply secret
+    inputs:
+      connectionType: Kubernetes Service Connection
+      kubernetesServiceEndpoint: hellman-mlops-deploy
+      command: apply
+      arguments: -f k8s/secret.yaml
+
+  - task: replacetokens@3
+    # Using replace tokens for the postgres data source conexions 
+    inputs:
+      targetFiles: "helm/values.yaml"
+      encoding: "auto"
+      writeBOM: true
+      actionOnMissing: "error"
+      keepToken: false
+      tokenPrefix: "$("
+      tokenSuffix: ")"
+
+  - task: HelmDeploy@0
+    displayName: helm upgrade grafana
+    inputs:
+      connectionType: 'Kubernetes Service Connection'
+      kubernetesServiceEndpoint: hellman-mlops-deploy
+      command: upgrade
+      # chartType: 'Name'
+      # chartName: 'grafana/grafana'
+      chartType: 'FilePath'
+      chartPath: 'helm/grafana-6.51.2.tgz'
+      namespace: $(NAMESPACE)
+      valueFile: helm/values.yaml
+      arguments: '--set admin.existingSecret=grafana-password'
+      releaseName: grafana
+
+  - task: replacetokens@3
+    # Using replace tokens for the postgres data source conexions 
```

Comment:
This comment doesnt corresponds to the task

Reply:
Good eye. Removed